osbs/build/spec.py

```diff
@@ -147,6 +147,7 @@ class CommonSpec(BuildTypeSpec):
     name = BuildIDParam()
     yum_repourls = BuildParam("yum_repourls")
     use_auth = BuildParam("use_auth", allow_none=True)
+    build_image = BuildParam('build_image')
```

Comment:
Could use 'default=' here, for consistency with how `git_ref` is handled. (Or is `git_ref` the one that needs bringing into line?)